src/main/scala/mesosphere/marathon/MarathonSchedulerActor.scala

```diff
@@ -524,7 +524,7 @@ class SchedulerActions(
     * command, and constraints) are applied consistently across running
     * application instances.
     */
-  private def update(
+  private def update( // linter:ignore UnusedParameter
```

Comment:
instead of a linter comment, can't we use `_` for a parameter name if it's unused?